easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
we never settled that discussion, I think? this is more about keeping ncurses easyconfigs consistent